src/server/api/controllers/getAllUsers.controller.js

```diff
@@ -0,0 +1,16 @@
+const knex = require('../../config/db');
+
+const getAllUsers = async () => {
+  try {
+    return await knex('game_scores')
+      .select('game_scores.fk_user_id', 'game_scores.score')
```

Comment:
You need SELECT statement that retrieves all the users that played a games (game_instance) and calculate their points in order to sort them. So we need:
1) username ( There is no name)
2) their total scores 👍
3) Sorting according to scores 👍

Reply:
Done! the name and scores can be shown now :)